concepticondata/conceptlists/Koelle-1854-335.tsv

```diff
@@ -0,0 +1,336 @@
+Koelle-1854-335-263	263	Salt	1274	SALT
+Koelle-1854-335-264	264	Sand	671	SAND
+Koelle-1854-335-265	265	Scorpion	1538	SCORPION
+Koelle-1854-335-266	266	Serpent	730	SNAKE
+Koelle-1854-335-267	267	Seven	1704	SEVEN
+Koelle-1854-335-268	268	Seventeen	2453	SEVENTEEN
+Koelle-1854-335-269	269	Sheep	1331	SHEEP
+Koelle-1854-335-270	270	Shirt	1622	SHIRT
+Koelle-1854-335-271	271	Shoe	1381	SHOE
+Koelle-1854-335-272	272	Shoulder	1482	SHOULDER
+Koelle-1854-335-273	273	Sick	1847	SICK
+Koelle-1854-335-274	274	silver	759	SILVER
+Koelle-1854-335-275	275	Six	1703	SIX
+Koelle-1854-335-276	276	Sixteen	2454	SIXTEEN
+Koelle-1854-335-277	277	Skin	763	SKIN
+Koelle-1854-335-278	278	Small-Pox	1054	SMALLPOX
+Koelle-1854-335-279	279	Smoke	778	SMOKE (EXHAUST)
+Koelle-1854-335-280	280	Soap	788	SOAP
+Koelle-1854-335-281	281	Son	1620	SON
+Koelle-1854-335-282	282	Soup	1547	SOUP
+Koelle-1854-335-283	283	Spear	945	SPEAR
+Koelle-1854-335-284	284	Spider	843	SPIDER
+Koelle-1854-335-285	285	Spoon	1378	SPOON
+Koelle-1854-335-286	286	Spoon (of wood)	1378	SPOON
+Koelle-1854-335-287	287	Stone	857	STONE
+Koelle-1854-335-288	288	Straight	1404	STRAIGHT
+Koelle-1854-335-289	289	Stranger	791	STRANGER
+Koelle-1854-335-290	290	Stupid	1518	STUPID
+Koelle-1854-335-291	291	Sun	1343	SUN
+Koelle-1854-335-292	292	Sword	1535	SWORD
+Koelle-1854-335-293	293	Ten	1515	TEN
+Koelle-1854-335-294	294	The large red-headed Lizard		
+Koelle-1854-335-295	295	The large red-headed Lizard (female)		
+Koelle-1854-335-296	296	Thigh	800	THIGH
+Koelle-1854-335-297	297	Thirteen	1708	THIRTEEN
+Koelle-1854-335-298	298	Thread	1161	THREAD
+Koelle-1854-335-299	299	Three	492	THREE
+Koelle-1854-335-300	300	Throat	1346	THROAT
+Koelle-1854-335-301	301	Toad	894	TOAD
+Koelle-1854-335-302	302	To-day	1283	TODAY
+Koelle-1854-335-303	303	Toe	1389	TOE
+Koelle-1854-335-304	304	Toe (the big one)		
+Koelle-1854-335-305	305	To-morrow	1329	TOMORROW
+Koelle-1854-335-306	306	Tongue	1205	TONGUE
+Koelle-1854-335-307	307	Tooth	1380	TOOTH
+Koelle-1854-335-308	308	Town (Village)	3609	TOWN OR VILLAGE
+Koelle-1854-335-309	309	town (walled)		
+Koelle-1854-335-310	310	Tree	906	TREE
+Koelle-1854-335-311	311	Trousers	809	TROUSERS
+Koelle-1854-335-312	312	Twelve	1707	TWELVE
+Koelle-1854-335-313	313	Twenty	1710	TWENTY
+Koelle-1854-335-314	314	Two	1498	TWO
+Koelle-1854-335-315	315	Vein	1924	VEIN
+Koelle-1854-335-316	316	Vein (rope)		
+Koelle-1854-335-317	317	Village	930	VILLAGE
+Koelle-1854-335-318	318	Waist-cloth		
+Koelle-1854-335-319	319	Walking-stick	1296	WALKING STICK
+Koelle-1854-335-320	320	War	935	WAR
+Koelle-1854-335-321	321	Wasp	1517	WASP
+Koelle-1854-335-322	322	Water	948	WATER
+Koelle-1854-335-323	323	Water (fresh)		
+Koelle-1854-335-324	324	Well	954	WELL
+Koelle-1854-335-325	325	Wet	1726	WET
+Koelle-1854-335-326	326	White	1335	WHITE
+Koelle-1854-335-327	327	White Man	2839	WHITE MAN
+Koelle-1854-335-328	328	witch	824	WITCH
+Koelle-1854-335-329	329	Woman	962	WOMAN
+Koelle-1854-335-330	330	Yam	410	YAM
+Koelle-1854-335-331	331	Yam (wild)	410	YAM
```

Comment:
YAM is mapped twice.

Reply:
As LEAF and others above. 331 could be dropped if we decide for a single mapping.